src/binder/binder_context.cpp

```diff
@@ -29,16 +29,19 @@ void BinderContext::AddRegularTable(parser::TableRef *table_ref,
   table_ref->TryBindDatabaseName(default_database_name);
   auto table_alias = table_ref->GetTableAlias();
   AddRegularTable(table_ref->GetDatabaseName(), table_ref->GetTableName(),
+                  table_ref->GetSessionNamespace(), table_ref->GetNamespace(),
                   table_alias, txn);
 }
 
 void BinderContext::AddRegularTable(const std::string db_name,
                                     const std::string table_name,
+                                    const std::string session_namespace,
```

Comment:
New arguments should be added last and assigned default values to keep previous tests working well.